src/actions/addtodo.js

```diff
@@ -0,0 +1,19 @@
+import {ADD_FAILED,ADD_SUCCESS,ADD_REQUEST} from '../contants'
+export const addRequest = (todo) => {
+    return {
+      type: ADD_REQUEST,
+      payload:todo
```

Comment:
```suggestion
      payload,
```
Mấy cái payload khác mình cũng làm tương tự em nhé